v23.1/export.md

```diff
@@ -15,7 +19,7 @@ Using the [CockroachDB distributed execution engine](architecture/sql-layer.html
 If you do not need distributed exports, you can [export tabular data in CSV format](#non-distributed-export-using-the-sql-client).
 
 {{site.data.alerts.callout_info}}
-`EXPORT` no longer requires an {{ site.data.products.enterprise }} license.
```

Comment:
I'd almost reverse the order of these callouts since if we recommend something else we could probably say that first thing

also I think this could be "green" (tip) or even "Red" if you wanna go hardcore ("DEFINITELY don't use this")

Reply:
Agree, switched these round. I have kept it as a note for now.... I felt very in between whether this should be a tip or warning, and so kept it boring with a note. I'll keep track of how the messaging around export vs changefeeds involves and maybe upgrade to a warning if it moves that way.